configs/simplecopypaste/mask_rcnn_r50_fpn_syncbn-all_rpn-2conv_ssj_270k_coco.py

```diff
@@ -0,0 +1,21 @@
+_base_ = [
```

Comment:
if we use more than 8 GPUS, may need to add `GPUSxbs ` in the config file name. Please @ZwwWayne have a decision